lib/spack/spack/package_base.py

```diff
@@ -114,6 +119,11 @@ def deprecated_version(pkg, version):
     return False
 
 
+def package_directory(cls):
+    """Returns the path to the package class directory."""
+    return os.path.abspath(os.path.dirname(cls.module.__file__))
```

Comment:
Why was extracting this function necessary? I think `package_dir` would work on package classes too.

Reply:
Can't recall.  Let me look into it.

Reply:
Removed